pages/common/dokku.md

```diff
@@ -0,0 +1,23 @@
+# Dokku
+
+> Docker powered mini-heroku
```

Comment:
"Docker-powered". Also, "mini-heroku" is not very self-explanatory; I would suggest expanding a little on what that means.